### ISSUE TITLE
Fix encoding in docker package database

### ISSUE THREAD

**Issue by jberkus** [pipelinedb/pipelinedb#1067]
The current docker packaging creates a SQL-ASCII installation.  We want UTF8.

**Comment by jberkus** [comment]
Fixed in commit e2e9bb75cf6953682779f28ba0e70a3cbf480811

Please check on dockerpkg branch, and merge to master.

**Comment by usmanm** [comment]
Thanks for the fixes! I'll merge the changes in right now. 

You might also want to hard reset your master branch to origin master (or just re-clone it). I think I rebased all of our commits after you cloned the repo so your `dockerpkg` branch says: *This branch is 286 commits ahead, 303 commits behind master*

```
git checkout master
git pull
git reset --hard origin/master
```
